lib/github_notification_monitor.rb

```diff
@@ -148,6 +149,15 @@ def remove_labels(command_value, author, issue)
     end
   end
 
+  def move_issue(command_value, author, issue)
+    match = /(?:([\w-]+)\/)?([\w-]+)/.match(command_value)
+    fq_repo_name = "#{match[1] || issue.organization_name}/#{match[2]}"
```

Comment:
Also, I'm wondering if it makes more sense to just not use regexes, and assume that the first `/` we see is a delimiter, then this code can be simplified to:

```ruby
org, repo = command_value.split("/", 2).unshift(issue.organization_name).last(2)
fq_repo_name = "#{org}/#{repo}"
```